.github/workflows/update-third-party-notices.yml

```diff
@@ -0,0 +1,27 @@
+name: Check and Update Third Party Notices
+on: workflow_call
+
+jobs:
+  license-check:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          registry-url: "https://registry.npmjs.org"
+      - run: npm install -g generate-license-file@1
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+          token: ${{ secrets.BOT_REPO_SCOPED_TOKEN }}
+      - name: Generate third party notices for studio
+        run: cd packages/studio; ../../generate-third-party-notices.sh
```

Comment:
can we use `working-directory` instead of manually cd-ing into the package?

Reply:
yep!